src/viam/examples/modules/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+include(CheckIncludeFile)
```

Comment:
This can also be backed out now that I'm using `find_path` / `find_library` below.